l10n_nl_base_ubl/tests/test_ubl_at_install.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2020 Onestein (<https://www.onestein.eu>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from lxml import etree
+from mock import patch
+
+from odoo.tests.common import Form, HttpCase
+
+
+class TestUblInvoice(HttpCase):
+    def setUp(self):
+        super().setUp()
+        with Form(
+            self.env["account.move"].with_context(default_type="out_invoice")
+        ) as invoice_form:
+            invoice_form.partner_id = self.env.ref("base.res_partner_4")
+            invoice_form.partner_id.country_id = self.env.ref("base.nl")
+            with invoice_form.invoice_line_ids.new() as invoice_line_form:
+                invoice_line_form.product_id = self.env.ref("product.product_product_4")
+                invoice_line_form.name = "product that cost 100"
+                invoice_line_form.quantity = 1
+                invoice_line_form.price_unit = 100.0
+            self.invoice = invoice_form.save()
+
+    def test_01_ubl_emulate_kvk_found(self):
+        nsmap, ns = self.env["base.ubl"]._ubl_get_nsmap_namespace("Invoice-1")
```

Comment:
I see no asserts in this test. Is this just checking that methods do not error out?